<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAttribWidget</class>
 <widget class="QGroupBox" name="CAttribWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Constant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scheme</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QWidget" name="schemeEditWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="editPushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userParamPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>U</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bankButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QWidget" name="schemeWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="schemeComboBox"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="srcLabel">
          <property name="text">
           <string>Src:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="srcComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Position</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverse Mass</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Speed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Random</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>User param</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LOD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Square LOD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Clamped LOD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Camped square LOD</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="NLQT::CBasicEditWidget" name="constAttribPlaneWidget" native="true"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="NLQT::CColorEditWidget" name="constRGBAWidget" native="true"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="NLQT::CEditRangeIntWidget" name="constRangeIntWidget" native="true"/>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="NLQT::CEditRangeFloatWidget" name="constRangeFloatWidget" native="true"/>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="NLQT::CEditRangeUIntWidget" name="constRangeUIntWidget" native="true"/>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="inputLabel">
     <property name="text">
      <string>Input mutliplier:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="3">
    <widget class="QCheckBox" name="clampCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Clamp</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="NLQT::CEditRangeFloatWidget" name="inMultiplierWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLQT::CEditRangeUIntWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CEditRangeFloatWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CBasicEditWidget</class>
   <extends>QWidget</extends>
   <header>basic_edit_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CColorEditWidget</class>
   <extends>QWidget</extends>
   <header>color_edit_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CEditRangeIntWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
